src/components/Header/Header.jsx

```diff
@@ -38,8 +39,8 @@ export const Header = () => {
       >
         <Flex align="center" justify="flex-start">
           <LogoWalletIcon width="45" height="45" color={iconsThemeColor} />
-          <Heading as="h1" mr="25px" ml="5px" size="lg">
-            BudgetTracker
+          <Heading as="h1" mr="25px" ml="5px" size="lg" color={headerTextColor}>
```

Comment:
```suggestion
          <Heading as="h1" mr="25px" ml="20px" size="lg" color={headerTextColor}>
```

It will be looking a little bit nicer with these settings.

Before:
![Screenshot 2022-07-21 at 13 14 49](https://user-images.githubusercontent.com/68731150/180201065-ecf55ee5-1b21-4ecb-9d99-463697a0f4d3.png)

After:
![Screenshot 2022-07-21 at 13 17 49](https://user-images.githubusercontent.com/68731150/180201324-c19cbd57-2e1f-4769-aad2-787cabe4fdec.png)